wordpress-mobile/WordPress-iOS
# [Stats Loading] Display cached data if there's no connection

This PR removes the NRV from the `StatsViewController` if there's no connection only if the feature flag is enabled. Cached data will be displayed instead (if it does exist).

## To test:
• Install a fresh version of the App
• Put the phone in airplane mode 
• Select stats. You should see the NRV (no connection)
• Tap back and switch off the airplane mode
• Select Stats and leave the insights load the data
• Tap back and put the phone in airplane mode
• Select Stats and you should see the cached data

Update release notes:
